com.unity.shadergraph/Editor/Data/Util/KeywordUtil.cs

```diff
@@ -65,11 +66,57 @@ public static string ToDeclarationString(this KeywordDefinition keywordDefinitio
             }
         }
 
+        public static bool NeedsMultiStageDefinition(this KeywordDescriptor keyword, ref List<KeywordShaderStage> dstStages)
+        {
+            dstStages.Clear();
+
+            // All doesn't need special treatment.
+            if (keyword.stages == KeywordShaderStage.All)
+                return false;
+
+            if ((keyword.stages & KeywordShaderStage.Vertex) == KeywordShaderStage.Vertex)
+                dstStages.Add(KeywordShaderStage.Vertex);
+            if ((keyword.stages & KeywordShaderStage.Hull) == KeywordShaderStage.Hull)
+                dstStages.Add(KeywordShaderStage.Hull);
+            if ((keyword.stages & KeywordShaderStage.Domain) == KeywordShaderStage.Domain)
+                dstStages.Add(KeywordShaderStage.Domain);
+            if ((keyword.stages & KeywordShaderStage.Geometry) == KeywordShaderStage.Geometry)
+                dstStages.Add(KeywordShaderStage.Geometry);
+            if ((keyword.stages & KeywordShaderStage.RayTracing) == KeywordShaderStage.RayTracing)
+                dstStages.Add(KeywordShaderStage.RayTracing);
+
+            return dstStages.Count > 1;
+
+        }
+
+        // TODO: BROKEN FIX!!!!
```

Comment:
what is broken? :)

Reply:
Oops this escaped my cleanup :D Will remove immediately!